src/DomainLib/Aggregates/ApplyEventRouter.cs

```diff
@@ -14,11 +14,15 @@ public class ApplyEventRouter<TAggregateRoot, TDomainEventBase>
     {
         private readonly IReadOnlyDictionary<Type, ApplyEvent<TAggregateRoot, TDomainEventBase>> _routes;
 
-        public ApplyEventRouter(IEnumerable<KeyValuePair<Type, ApplyEvent<TAggregateRoot, TDomainEventBase>>> routes)
+        public ApplyEventRouter(IEnumerable<KeyValuePair<Type, ApplyEvent<TAggregateRoot, TDomainEventBase>>> routes, 
+                                IEventNameMapping eventNameMappings)
         {
             _routes = ImmutableDictionary.CreateRange(routes);
+            EventNameMappings = eventNameMappings;
         }
 
```

Comment:
Can we rename the interface to `IEventNameMap`, and the implementation to `EventNameMap`? Then the property here can also be called `EventNameMap`. Seems a bit more concise and consistent.

Reply:
Renamed